### ISSUE TITLE
Binary file attributes require full system path (when using simplified attribute in yml)

### ISSUE THREAD

**Issue by seiji-okamoto-informa** [kaliop-uk/ezmigrationbundle#74]
According to the documentation, binary files should be placed in MigrationVersions/files/, and paths to these files should be relative.  However, this generates the following error:

> Migration aborted! Reason: Error in execution of step 1: Argument '$value->inputUri' is invalid: 'data_and_forecast_catalogue.xlsx' is wrong value in class 'eZ\Publish\Core\FieldType\BinaryFile\Type' in file /srv/www/www.ovumkc.com/vendor/ezsystems/ezpublish-kernel/eZ/Publish/Core/FieldType/BinaryBase/Type.php line 97

Setting the full path from / does work.

**Comment by seiji-okamoto-informa** [comment]
Further investigation shows a relative path from the site root works as well:

```yml
attributes:
    - name: 'My file name'
    - file: 'src/Ovum/SiteBundle/MigrationVersions/files/my_file.xls'
```

**Comment by gggeek** [comment]
If you read the docs, you will see that the syntax officially supported at the moment (version 2) for files is:

````
attributes:
    - name: 'My file name'
    - file:
        path:  'my_file.xls'
````

This will be better supported in version 3, where the simplified syntax you are already using will be supported officially

**Comment by gggeek** [comment]
Fixed in 3.0beta3
